R/utils.R

```diff
@@ -9,3 +9,103 @@ get_bs_version <- function() {
     "3"
   }
 }
+
+#' Panel group widget
+#' @md
+#'
+#' @description `r lifecycle::badge("experimental")`
+#' @param title (`character`)\cr title of panel
+#' @param ... content of panel
+#' @param collapsed (`logical`, optional)\cr
+#'  whether to initially collapse panel
+#' @param input_id (`character`, optional)\cr
+#'  name of the panel item element. If supplied, this will register a shiny input variable that
+#'  indicates whether the panel item is open or collapsed and is accessed with `input$input_id`.
+#'
+#' @return (`shiny.tag`)
+#'
+#' @keywords internal
+panel_item <- function(title, ..., collapsed = TRUE, input_id = NULL) {
```

Comment:
hmm - this seems quite a lot of code to take from I assume teal.widgets? Not sure about this

Reply:
There are a few solutions like add shinyBS package dependency to both packages or add teal.widgets dependency to teal.reporter. Both will increase the number of dependencies for our packages.

WHen add teal.widgets here:
```
> setdiff(pacs::pac_deps_user("teal.widgets", local = T)[,1], pacs::pac_deps_user("teal.reporter", local = T)[,1])
 [1] "colorspace"   "fansi"        "farver"       "formatters"   "ggplot2"     
 [6] "gtable"       "isoband"      "labeling"     "lattice"      "MASS"        
[11] "Matrix"       "mgcv"         "munsell"      "nlme"         "pillar"      
[16] "pkgconfig"    "purrr"        "R.cache"      "R.methodsS3"  "R.oo"        
[21] "R.utils"      "RColorBrewer" "rematch2"     "rprojroot"    "rtables"     
[26] "scales"       "shinyjs"      "styler"       "tibble"       "utf8"        
[31] "vctrs"        "viridisLite"
```

Reply:
So I will recommend to copy paste this function.

Reply:
yeah I guess 😞